FirebaseSwiftUI/FirebaseFacebookSwiftUI/Sources/Services/FacebookProviderAuthUI.swift

```diff
@@ -109,7 +119,7 @@ public class FacebookProviderAuthUI: FacebookProviderAuthUIProtocol {
     if let idToken = AuthenticationToken.current {
       let credential = OAuthProvider.credential(withProviderID: providerId,
                                                 idToken: idToken.tokenString,
-                                                rawNonce: rawNonce)
+                                                rawNonce: rawNonce!)
```

Comment:
It's not immediately clear from the code that this optional will never be `nil` here, so maybe add a comment explaining why it's safe.

Reply:
I've updated code to guard the rawNonce instead as I think that is better: [`38a1b0c` (#1256)](https://github.com/firebase/FirebaseUI-iOS/pull/1256/commits/38a1b0c6d3627673e4c590501a726ff575583cc2)